Setup/InstallSchema.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Klaviyo\Reclaim\Setup;
+use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+class InstallSchema implements InstallSchemaInterface
+{
+    /**
+     * {@inheritdoc}
+     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+     */
+    public function install(SchemaSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $installer = $setup;
+        $installer->startSetup();
+
+        $eavTable1 = $installer->getTable('quote');
+        $eavTable2 = $installer->getTable('sales_order');
+
+        $columns = [
+            'kl_sms_consent' => [
+                'type' => \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
```

Comment:
Unless I am mistaken that these values are not just true/false .  why are you using text type instead of smallint(5) teh convention used in Magento MySql for everything else representing a boolean?

![image](https://user-images.githubusercontent.com/38661954/110821288-ec1fab80-825d-11eb-9a73-289a9a11503c.png)

Reply:
totally, should be a bool 👍

Reply:
leaving as a textfield due to tiny int's on different mysql versions, it also allows us to potentially add more in the future?